### PR TITLE
Share card

### PR DESCRIPTION
**Pull request by janbaykara**

### CODE REVIEW

File: src/components/seo.tsx [hunk 1]
```diff
@@ -58,9 +67,25 @@ function SEO({ description, lang, meta, title }) {
 					property: `og:type`,
 					content: `website`,
 				},
+				{
+					property: `og:url`,
+					content: _url,
+				},
+				{
+					property: "og:image",
+					content: `${site.siteMetadata.siteUrl}/sharecard.jpg`
```
Comment (conatus) [inline src/components/seo.tsx]: We might want to customise this in future, but think this is a good start.

**Comment by conatus** [comment]
I tried this with the [Twitter Card Validator](https://cards-dev.twitter.com/validator) and it didn't seem to work.

Screenshot 2020-02-11 at 12 23 09

Facebook [looks good however for the site](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fdeploy-preview-21--commonknowledge.netlify.com) and for a [blog page](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fdeploy-preview-21--commonknowledge.netlify.com%2Fwriting%2Fweeknotes-8) works less well.

**Comment by conatus** [comment]
Can you also pass through the little description from the blog posts to the `description` tag?

**Comment by janbaykara** [comment]
We're going to have to try this on a merge because I think there's an issue with the referenced URL hosts not being the same as the preview URL.

**Pull request merged**